right_column.md

```diff
@@ -75,29 +75,39 @@ Python.
   [text instructions](https://coderefinery.github.io/installation/git/#configuring-git),
   [video](https://www.youtube.com/watch?v=WdDTp8NeHBs&t=258s).
 
+
+### Host
+- Naoe Tatara
+
 ### Instructors
 
 - Radovan Bast
 - Anne Fouilloux
 - Bjørn Lindi
 - Thor Wikfeldt
 - Stefan Negru
+- Richard Darst
```

Comment:
```suggestion
```
Technically I'm not an instructor... what if we renamed the above instruction to "Organizers" and put Naoe and I there?

Reply:
Right, I overlooked where your name was in the schedule ;)